src/content/guides/using-persistent-volumes-on-aws-with-efs/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Using Persistent Volumes on AWS with EFS"
+description = "Tutorial on how to use dynamically provisioned Persistent Volumes on a cluster running on Amazon Web Services with EFS"
+date = "2020-02-06"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+# Using Persistent Volumes on AWS with EFS
+
+If your cluster is running in the cloud on Amazon Web Services (AWS) the most common way to store data is using EBS volumes with the [dynamic provisioner](https://docs.giantswarm.io/guides/using-persistent-volumes-on-aws/). Sometimes EBS is not the optimal solution.
+
+The advantages of using EFS over EBS are:
+- EFS data can be accessed from all Availability Zone in the same region while EBS is tied to a single Availability Zone.
+- EFS has the capability to mount the same Persistent Volume to multiple pods at the same time using the ReadWriteMany [access mode](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).
+- EFS will not hit the [AWS Instance Volume Limit](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html) as it is a software mount and will avoid the [Impaired EBS](https://docs.giantswarm.io/guides/aws-impaired-volumes/) issue.
+- EFS mount times are better than EBS.
+
+If you need to use EFS to provision volumes, be advised:
+- All Kubernetes Persistent Volumes will be stored in the same EFS instance. You can deploy multiple provisioners per cluster, each having its own storage-class and EFS instance.
+- [EFS throughtput](https://docs.aws.amazon.com/efs/latest/ug/performance.html) need to be set up accordingly in order to not have performance issues. We only recommend Provisioned Throughput, and if you need high performance you will need EBS.
+- EFS backups are done with [AWS Backup](https://aws.amazon.com/backup/) and it does not have the snapshot feature of EBS.
+- You cannot limit the amount of data stored in an EFS volume. The requested value in Kubernetes is ignored.
+- EFS mount targets are limited to 1 subnet per Availability Zone. Each NodePool will create a different subnet per AZ, plan accordingly. 
+
+
+## Provision an EFS instance on AWS
+
```

Comment:
- One line of intro for this section would be nice, like further down in the `Installing EFS Provisioner` section.
- Would the list make more sense as a numbered list instead of bullets? For step-by-step things that makes it more clear that the order is important.

Reply:
Added introduction

Reply:
Added introduction and numbered lists